### PR TITLE
Admin portal data changes

### PR DESCRIPTION
**Pull request by BradHN1**
Issue #415 - consolidate data input in one form

### CODE REVIEW

File: v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js [hunk 1]
```diff
@@ -252,138 +252,138 @@ class HomePageEditForm extends Component {
...
-                    label: 'Goal for Carbon Footprint Reduction (lbs.)',
-                    placeholder: 'eg. 10000000',
-                    fieldType: 'TextField',
-                    contentType: 'number',
-                    isRequired: false,
-                    defaultValue: goal && goal.target_carbon_footprint_reduction,
-                    dbName: 'target_carbon_footprint_reduction',
-                    readOnly: false
-                  },
+                  //{
```
Comment (Opoku-Agyemang) [inline v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js:255]: can we just delete the code?

File: v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js [hunk 2]
```diff
@@ -63,13 +63,13 @@ class HomePageEditForm extends Component {
...
     }
 
-    const communityResponse = await apiCall('/communities.info', { community_id: id });
-    if (communityResponse && communityResponse.data) {
-      await this.setStateAsync({ goal: communityResponse.data.goal });
-    } else {
-      await this.setStateAsync({ noDataFound: true });
-      return;
-    }
+    //const communityResponse = await apiCall('/communities.info', { community_id: id });
```
Comment (Opoku-Agyemang) [inline v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js:66]: can we just delete?

**Review by Opoku-Agyemang — COMMENTED** [review]
LGTM but we can delete the commented code

**Comment by Opoku-Agyemang** [comment]
Also can you add some screenshots with this?

File: v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js (hunk 1 above)
Comment (BradHN1) [reply v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js]: Done

File: v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js (hunk 2 above)
Comment (BradHN1) [reply v2.0/app/containers/MassEnergizeSuperAdmin/Pages/Home.js]: Done

**Pull request merged**